src/wp-admin/install.php

```diff
@@ -229,11 +66,13 @@ function display_setup_form( $error = null ) {
  */
 global $wp_version, $required_php_version, $required_mysql_version;
 
-$php_version    = phpversion();
-$mysql_version  = $wpdb->db_version();
-$php_compat     = version_compare( $php_version, $required_php_version, '>=' );
-$mysql_compat   = version_compare( $mysql_version, $required_mysql_version, '>=' ) || file_exists( WP_CONTENT_DIR . '/db.php' );
+// Get and check for sufficient versions of PHP and MySQL.
+$php_version   = phpversion();
+$mysql_version = $wpdb->db_version();
+$php_compat    = version_compare( $php_version, $required_php_version, '>=' );
+$mysql_compat  = version_compare( $mysql_version, $required_mysql_version, '>=' ) || file_exists( WP_CONTENT_DIR . '/db.php' );
 
+// Insufficient PHP and/or MySQL? Set a flag with an error message.
 if ( !$mysql_compat && !$php_compat ) {
 	/* translators: 1: ClassicPress version number, 2: Minimum required PHP version number, 3: Minimum required MySQL version number, 4: Current PHP version number, 5: Current MySQL version number */
 	$compat = sprintf( __( 'You cannot install because <a href="https://codex.wordpress.org/Version_%1$s">ClassicPress %1$s</a> requires PHP version %2$s or higher and MySQL version %3$s or higher. You are running PHP version %4$s and MySQL version %5$s.' ), $wp_version, $required_php_version, $required_mysql_version, $php_version, $mysql_version );
```

Comment:
Let's update these strings too.  Right now they indicate the WordPress-equivalent version (currently 4.9.8) and link to the WP codex.

The logic for determining the link to the ClassicPress version can work as follows:

- If `classicpress_version()` does not contain a `+` character, link to `https://github.com/ClassicPress/ClassicPress-release/releases/%1$s` (e.g. https://github.com/ClassicPress/ClassicPress-release/releases/1.0.0-beta1).
- If `classicpress_version()` contains a `+` character, there are several possibilities for why, not all of which actually have an official release build.  In that case we should probably just link to the list at https://github.com/ClassicPress/ClassicPress-release instead.

Reply:
👍

Reply:
Updated.